src/BoundaryConditions/fill_halo_regions.jl

```diff
@@ -191,22 +191,22 @@ fill_first(bc1, bc2)               = true
 ##### Double-sided fill_halo! kernels
 #####
 
-@kernel function _fill_west_and_east_halo!(c, west_bc, east_bc, loc, grid, args) 
+@kernel function _fill_west_and_east_halo!(c, west_bc, east_bc, loc, grid, pressure_corrected, args) 
```

Comment:
Can we implement this differently? I think we can do it in a simpler way that doesn't require a flag to be propagated